.vscode/c_cpp_properties.json

```diff
@@ -1,53 +1,31 @@
 {
     "configurations": [
         {
-            "name": "Linux",
+            "name": "Win32",
```

Comment:
Don't change the existing configuration. Add a new one under `Win32`, and leave `Linux` alone.